app/models/claims/claim_window.rb

```diff
@@ -25,6 +25,7 @@ class Claims::ClaimWindow < ApplicationRecord
   default_scope -> { kept }
 
   belongs_to :academic_year
+  has_many :claims, class_name: "Claims::Claim"
```

Comment:
We don't really have a need for this just yet. It might be better to leave out this code until we have a use for it.